.travis.yml

```diff
@@ -6,6 +6,7 @@ before_script: |
   rustup component add rustfmt-preview &&
   rustup component add clippy-preview
 script: |
+  cargo fmt -- --write-mode diff &&
```

Comment:
I believe they changed this to be `-- --change`

Reply:
`--check`apparently!

Reply:
Ah! my bad haha